core/dovecot/start.py

```diff
@@ -9,7 +9,7 @@
 from podop import run_server
 from socrate import system, conf
 
-system.set_env(log_filters=r'waitpid\(\) returned unknown PID \d+$')
+system.set_env()
```

Comment:
Can you enhance this to filter out the SSL context warnings?
```
dimitri@mailutest:/mailu$ docker compose logs imap | grep SSL
mailu-imap-1  | Apr 22 08:15:31 lmtp(78): Error: SSL context initialization failed, disabling SSL: Can't load SSL certificate (ssl_cert setting): The certificate is empty
mailu-imap-1  | Apr 22 08:26:31 lmtp(274): Error: SSL context initialization failed, disabling SSL: Can't load SSL certificate (ssl_cert setting): The certificate is empty
```

```suggestion
system.set_env(log_filters=[
    r'Error\: SSL context initialization failed, disabling SSL: Can\'t load SSL certificate \(ssl_cert setting\)\: The certificate is empty$',
])
```

Reply:
Done in [7e46e14](https://github.com/Mailu/Mailu/pull/2794/commits/7e46e1491eede46cc04dd74d61973f829fabb10b)